next-app/src/routes/password/changed.svelte

```diff
@@ -0,0 +1,34 @@
+<script context=module>
+	export const prerender = true
```

Comment:
There might be an issue with prerendering pages like this one; the `getSession` hook might not run when the user clicks on links to the rest of the site. See https://github.com/sveltejs/kit/issues/4426. Though that bug will probably be solved by the time we deploy this.

Reply:
@rmunn nice job with your Svelte issue repro and getting the attention of the original dev! 💯

Reply:
I haven't really put any integrity checks in place on this page to ensure the user came from the correct previous page because if someone is fiddling with url's they can expect some weirdness :-) like a page that says they changed their password when they didn't...so maybe your warning about `getSession` might prove to be relevant for other future pages?

Reply:
Yes, it probably will... but I hope https://github.com/sveltejs/kit/issues/4426 will be resolved before we need to care about that.

As with the other threads, I'm going to resolve this conversation so it doesn't block our ability to merge the PR. That doesn't mean I don't want to continue discussing this, but it's probably better to do so in primary PR comments so that the PR merge isn't blocked. (Or we could consider whether the "All conversations must be resolved" setting is worth the extra hassle of doing this.)

Reply:
I still kind of like the resolve rules we have in place.  I'm adding the discussion items to our team meeting agenda